index.php

```diff
@@ -33,6 +35,7 @@
 		<ul>
 			<li><a href="https://www.mozilla.cz/">Mozilla.cz</a></li>
 			<li><a href="https://www.mozilla.cz/stahnout/">Stáhnout</a></li>
+                        <li><a href="http://www.seamonkey-project.org/donate/">Přispět</a></li>
```

Comment:
Vyřešil jsem to tak, že jsem mergnul (7333b5d) tvůj commit, který odebírá link na forum.mozilla.cz. Je to tak správně?